Makefile

```diff
@@ -0,0 +1,25 @@
+#
+# Makefile to run the business analytics software using metabase
+#
+
+#runs the docker-compose.yml showing logs
+up :
+	@echo "Starting Riff Business Analytics.";
+	@docker-compose up;
+	@echo "Processes Complete. Metabase running on port 3000.";
+
+#runs the docker-compose.yml detached
+up-detach :
+	@echo "Starting Riff Business Analytics.";
+	@docker-compose up --detach;
+	@echo "Processes Complete. Metabase running on port 3000.";
+
+#stops and removes containers
+down :
+	@docker-compose down
+
+#runs the script to restore database backups for metabase to run queries on
+restore :
+	@echo "Restoring mysql database backups from 'database-backups' folder.";
+	@bash bin/restore-database-backups.sh;
+	@echo "Processes Complete. Metabase running on port 3000.";
```

Comment:
I'm not sure what you mean by `Processes Complete` or how you can say for sure that metabase is running on port 3000 when you didn't do anything for this target to make that happen.

Reply:
Sorry that should have been removed from this target. The command used to be up-restore, which did both run docker-compose up, then do the DB restorations. But I found that it sometimes failed because the mysql container wasn't ready. I felt for the scope of the current task, it would be fine to just run `make up`, then after a few minutes, run `make restore` in a new tab. (Which the new tab part needs to be added to the documentation). I think there needs to be a more elegant way of determining when docker-compose finishes so we can @echo something on the screen, but as for the DB restorations, it seems to be successful in waiting for all sql backups to be restored, before hitting the final @echo 'Processes Complete'. Maybe a more descriptive line like 'Database restorations complete.', would be better?

Reply:
I'm not sure that `make restore` does a lot for you. Perhaps the doc should just tell you to run `bin/restore-database-backups.sh`.

Is the echo after the actual work really helpful for all of those targets? If the answer is yes, then great, but if not perhaps it should be removed.

As a 1 time thing you also could just ask that the user supply the container name as an argument to the script, and then you wouldn't have to name it in the docker-compose (although having a constant name does make things easier, it is only an occasional thing).

Reply:
Removed `make restore` and updated docs accordingly.